docker/test/performance-comparison/compare.sh

```diff
@@ -113,6 +113,9 @@ function restart
     # https://github.com/jemalloc/jemalloc/wiki/Getting-Started
     export MALLOC_CONF="confirm_conf:true"
 
+    # Temporary to measure effect GWP has on performance
+    export CLICKHOUSE_GWP_ASAN_OPTIONS="sample_rate=100,slot_size=512,max_simultaneous_allocations=512"
```

Comment:
I did not looked into performance tests deeply (will do this later), but seems that they are fine even with more higher sample rate.
Let's remove this line for now them and see how they will differs.